_articles/open-firmware-smmstore.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: article
+title: Clear the SMMSTORE ROM
+description: >
+  How to fix the failed with exit status: exit code: 5 message
```

Comment:
I think error code 5 should be in the title:

"Firmware Flash to Clear the SMMSTORE ROM (Error code 5)"